scripts/integration_test.sh

```diff
@@ -17,65 +17,98 @@
 # This script will start an isolated server and run the python API against it
 #
 
-# Install dependencies silently
-sudo snap install protobuf --classic 2>&1 > /dev/null
-sudo apt-get install python3-pip python3-setuptools python3-pip python3-dev python-setuptools-doc python3-wheel 2>&1 > /dev/null
-python3 -m pip install cython 2>&1 > /dev/null
-python3 -m pip install -r ./tests/PythonEthApi/requirements.txt 2>&1 > /dev/null
-
-# Need to build evm...
-git clone https://github.com/Zilliqa/evm-ds.git
-
-cd evm-ds
-cargo --version
-echo "building evm ds"
-
-cargo build --verbose --release --package evm-ds 2>&1 > /dev/null
-
-cd -
-
-# Just to check evm-ds has been built
-if [[ -d /home/travis ]]; then
-    ls /home/travis/build/Zilliqa/Zilliqa/evm-ds/target/release/evm-ds
-
-    # Modify constants.xml for use by isolated server
-    cp constants.xml constants_backup.xml
-    sed -i 's/.LOOKUP_NODE_MODE.false/<LOOKUP_NODE_MODE>true/g' constants.xml
-    sed -i 's/.ENABLE_EVM>.*/<ENABLE_EVM>true<\/ENABLE_EVM>/g' constants.xml
-    sed -i 's/.EVM_SERVER_BINARY.*/<EVM_SERVER_BINARY>\/home\/travis\/build\/Zilliqa\/Zilliqa\/evm-ds\/target\/release\/evm-ds<\/EVM_SERVER_BINARY>/g' constants.xml
-    sed -i 's/.EVM_LOG_CONFIG.*/<EVM_LOG_CONFIG>\/home\/travis\/build\/Zilliqa\/Zilliqa\/evm-ds\/log4rs.yml<\/EVM_LOG_CONFIG>/g' constants.xml
+# Firstly determine if the user is running this locally, or is this in the CI
+
+RUNNING_LOCALLY=1
```

Comment:
determine if running locally since users will want to run this script locally if it fails on CI